app/definitions/rest/v1/emojiCustom.ts

```diff
@@ -9,7 +9,7 @@ export type EmojiCustomEndpoints = {
 		} & PaginatedResult;
 	};
 	'emoji-custom.list': {
-		GET: (params: { query: string }) => {
+		GET: (params: { updatedSince: string }) => {
```

Comment:
are you sure about this?

Reply:
Not sure how it works on the web, but if we actually send a param called `query` instead of sending `updatedSince` then it crashes the request on startup.